fitnessllm_dataplatform/utils/task_utils.py

```
@@ -0,0 +1,13 @@
+"""Module for used task utilities."""
+import os
+
+
+def load_into_env_vars(options: dict):
```

Ruff (0.8.2)

Missing return type annotation for public function `load_into_env_vars`

Add return type annotation: `None`

(ANN201)